### DISCUSSION TITLE
results

### DISCUSSION THREAD

**Discussion by ferroteam** [Tomoki-YAMASHITA/CrySPY#57] [General]
Hello,
I performed a crystal search using RS method (1000 structures). There are 5 stages in the process. For some structures the search is completed (i.e., all 5 stages are done) and when I look into cryspy_rslt_energy_asc file I see the following:

     Spg_num     Spg_sym  Spg_num_opt Spg_sym_opt  E_eV_atom  Magmom      Opt
668       92    P4_12_12           92    P4_12_12  -3.187351     NaN  not_yet
646       62        Pnma           62        Pnma  -3.187223     NaN  not_yet
586       19  P2_12_12_1           19  P2_12_12_1  -3.185611     NaN  not_yet
390       92    P4_12_12           92    P4_12_12  -3.184457     NaN  not_yet
682       56        Pccn           56        Pccn  -3.183607     NaN  not_yet
809      110      I4_1cd          110      I4_1cd  -3.181608     NaN  not_yet
661      229       Im-3m          221       Pm-3m  -3.176966     NaN  not_yet
941      229       Im-3m          221       Pm-3m  -3.176839     NaN  not_yet
381      173        P6_3          173        P6_3  -3.175062     NaN  not_yet
841       64        Cmce           64        Cmce  -3.174583     NaN  not_yet
481      106      P4_2bc          106      P4_2bc  -3.173147     NaN  not_yet
200      161         R3c          161         R3c  -3.173107     NaN  not_yet
777      106      P4_2bc          106      P4_2bc  -3.172855     NaN  not_yet
......

I'm confused by the last column which says "not_yet" for Opt status. I remember to see in CrysPY_examples that it should read "done" if it is over. So, does it mean my calculations still not completed?

Thank you.

**Comment by Tomoki-YAMASHITA** [comment]
When you use the VASP code, CrySPY looks for "reached required accuracy" from OUTCAR in the final stage.
If it is found, CrySPY shows 'done' in cryspy_rslt. This is just a rough guide.  If you do not perform structure optimization in the final stage, "reached required accuracy" does not appear. So it depends on INCAR_5.

**Reply by USFmatscilab** [reply]
Thank you, Tomoki!
Yes, I use VASP code and last stage is just SCF calculation.

I was wondering, what is acceptable number of trial structures for RS method? Is 1000 enough? Or should I stick to EA or BO or LAQA, where only 1 stage is required?

Thanks!

**Reply by Tomoki-YAMASHITA** [reply]
Acceptable number of trial structures depends on the system; number of elements and atoms, potential energy surface, and so on.
For example, simple ionic crystals such as Na8Cl8 is easy, but Si16 (covalent) is more difficult.

If you cannot find stable structure with 1000 or 2000 trials in RS, better to use EA.
In EA and BO, we usually 2 or 3 stages like RS. LAQA is a slightly special method, so the stage is fixed at 1.
